pyccel/ast/numpyext.py

```diff
@@ -156,7 +177,8 @@ def __init__(self, arg):
         8 : PythonComplex,
         16 : NumpyComplex128,},
     'Bool':  {
-        4 : PythonBool}
+        4 : PythonBool,
+        4 : NumpyBool}
```

Comment:
can you please keep one casting function here, since they are almost the same and have the same precision?